Mathlib/Combinatorics/SimpleGraph/Copy.lean

```diff
@@ -177,18 +239,11 @@ protected alias IsContained.bot := bot_isContained_iff_card_le
 /-- A simple graph `G` contains all `Subgraph G` coercions. -/
 lemma Subgraph.coe_isContained (G' : G.Subgraph) : G'.coe ⊑ G := ⟨G'.coeCopy⟩
 
-/-- The isomorphism from `Subgraph A` to its map under a copy `Copy A B`. -/
-noncomputable def Subgraph.Copy.map (f : Copy A B) (A' : A.Subgraph) :
```

Comment:
It looks like you renamed this without deprecation, and it's not clear to me that your new name is any better: in fact it seems closer to the name you complained about in the past

Reply:
Yes, because I worked out again why I had chosen the name as such. The current name is problematic in that it lies in two namespaces: with `Subgraph.Copy.map`, you can neither do dot notation on the `Subgraph` nor on the `Copy`. Furthermore, I have discovered that the most important use case of `isoSubgraphMap` is when `A' := ⊤`, which I have made as `isoToSubgraph`, whose name is close to `isoSubgraphMap` and independently guessable.